Mage.Sets/src/mage/cards/m/MirrorBox.java

```diff
@@ -87,12 +87,6 @@ public boolean applies(GameEvent event, Ability source, Game game) {
 
 class MirrorBoxBoostEffect extends ContinuousEffectImpl {
 
-    private static final FilterPermanent filter = new FilterControlledCreaturePermanent();
-
-    static {
-        filter.add(TokenPredicate.FALSE);
-    }
-
```

Comment:
This is a `FilterControlledCreaturePermanent`, so it should be using the other static filter.